tests/integration/test_cmorizer.py

```diff
@@ -1,15 +1,16 @@
-"""Tests for the module :mod:`esmvaltool.cmorizers.obs.cmorize_obs`."""
+"""Tests for the module :mod:`esmvaltool.cmorizers.data.cmorize_obs`."""
 
 import contextlib
 import os
 import sys
 
+import pytest
 import iris
```

Comment:
I think you need to import all iris modules (iris.cube, iris.coords, iris.coord_systems) explicitely since the last release

Reply:
Updated